test/integration/visitor_cannot_view_orders_and_admin_views_test.rb

```diff
@@ -0,0 +1,94 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.